en/services/multi_gcs.md

```diff
@@ -0,0 +1,128 @@
+# Multi GCS protocol
+## Introduction
+A controlled vehicle should only accept MAVLink commands (or command-like messages) that are sent by the controlling GCS, or from other components with the same system id. Commands from other system should be rejected with MAV_RESULT_PERMISSION_DENIED (except for this command, which may be acknowledged with MAV_RESULT_ACCEPTED if control is granted).
```

Comment:
Generally the first mention (at least) of an XML entity should link to a more detailed definition. Ideally that is the message definition in _this_ doc, which then links to the main definition. In this case the new item isn't listed in the doc so i have linked direct to the common.xml definition.

```suggestion
A controlled vehicle should only accept MAVLink commands (or command-like messages) that are sent by the controlling GCS, or from other components with the same system id. Commands from other system should be rejected with [MAV_RESULT_PERMISSION_DENIED](../messages/common.md#MAV_RESULT_PERMISSION_DENIED) (except for this command, which may be acknowledged with [MAV_RESULT_ACCEPTED](../messages/common.md#MAV_RESULT_ACCEPTED) if control is granted).
```